core-java/src/test/java/com/baeldung/nulls/PrimitivesAndWrapperUnitTest.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.nulls;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class PrimitivesAndWrapperUnitTest {
+
+    @Test
+    public void givenWrappers_whenBothArgsNonNull_thenReturnResult() {
+
+        Integer sum = PrimitivesAndWrapper.wrapperSum(0, 0);
+
+        assertEquals(0, sum.intValue());
+    }
+
+    @Test()
+    public void givenWrappers_whenOneArgIsNull_thenThrowNullPointerException() {
+        assertThrows(NullPointerException.class, () -> PrimitivesAndWrapper.wrapperSum(null, 2));
+    }
+
+    @Test()
+    public void givenWrappers_whenBothArgsAreNull_thenThrowNullPointerException() {
```

Comment:
Use the suggestions above to come up with a new name for the rest of the test methods.